components/inputs/input-time.js

```diff
@@ -177,6 +180,9 @@ class InputTime extends SkeletonMixin(FormElementMixin(LitElement)) {
 				:host([hidden]) {
 					display: none;
 				}
+				:host([data-mobile]) .d2l-input-time-menu {
+					text-align: center;
```

Comment:
Centering the times in the mobile mode so they aren't left-adjusted. Otherwise, the times look really weird on wider screens with a bunch of empty space.

Reply:
I wonder if this is something menus (or in this case dropdown-menus) should be doing themselves? That way any menu that goes into a tray would have centred items. Maybe a question for Jeff.

Reply:
I agree with you on the text alignment, I'll ask Jeff about that. And maybe we might want it for bottom trays only, since left and right aren't supposed to be symmetrical anyways. Not sure about ignoring the maximum height for menus with trays, there may be some scenario where a consumer wants a short bottom tray on purpose? I'll start a discussion on slack 😄

Reply:
Cool sounds good!

Reply:
I am actually thinking to keep the centering here in input time rather than a menu default setting. I realized that the filter, which uses a menu, wants a left-centered list of items as per the spec. I think it makes sense to keep that center formatting in the time picker only for now.